### ISSUE TITLE
Not work in iphone

### ISSUE THREAD

**Issue by bravemantonyzheng** [kodyl/react-document-meta#13]
It can not work properly in iphone.

**Comment by danieljuhl** [comment]
Could you please provide some details, of you expect anyone to look into this. The short answer is that it does work.

**Comment by bravemantonyzheng** [comment]
sorry, below is my code.

```javascrpt
// routes.jsx
export default (
  <Route>
    <Route path="/" component={ App }>
      <Route path="dailyoperation" component={ DailyOperation } />
    </Route>
    <Route path="/test1" component={ Test1 }/>
    <Route path="/test2" component={ Test2 }/>
    <Route path="/test3" component={ Test3 }/>
  </Route>
);
```

```javascript
// Test2.jsx

import React, { Component, PropTypes } from 'react';
import DocumentTitle from 'react-document-title';
import { Link } from 'react-router';

class Test2 extends Component {
  render() {
    return (
      <DocumentTitle title="test2">
      <div>
        test2 <Link to="/test3">test3</Link>
      </div>
      </DocumentTitle>
    );
  }
}

export default Test2;
```

I test in In iphone5,6 wechat browser, the title does't change when i switch in different page.

**Comment by danieljuhl** [comment]
Sorry, but this is not a very useful test case (you've only included some of your code) - in my end, it works just fine, and I haven't heard of any problems.

Can you please make a FULL but minimal reproducible case.

**Comment by danieljuhl** [comment]
Should be fixed in `2.0.1`, otherwise please re-open and provide more details
